tapir/coop/tests/membership_resignation/test_delete_view.py

```diff
@@ -69,16 +76,20 @@ def test_membershipResignationDeleteView_default_sharesEndDateSetToNone(self):
 
     def test_membershipResignationDeleteView_default_logEntryCreated(self):
         self.given_feature_flag_value(feature_flag_membership_resignation, True)
-        self.login_as_member_office_user()
+        actor = self.login_as_member_office_user()
         resignation: MembershipResignation = MembershipResignationFactory.create()
-
         response = self.client.post(
-            reverse("coop:resign_member_remove", args=[resignation.id])
+            reverse("coop:resign_member_remove", args=[resignation.id]),
+            follow=True,
         )
         self.assertStatusCode(response, HTTPStatus.OK)
-
-        self.fail(
-            "No corresponding log entry class, see "
-            "TestMembershipResignationEditView.test_membershipResignationEditView_default_logEntryCreated "
-            "for an example on how to test log entries"
-        )
+        self.assertEqual(MembershipResignation.objects.all().count(), 0)
+        for shareownership in ShareOwnership.objects.filter(
+            share_owner=resignation.share_owner
+        ):
+            self.assertEqual(shareownership.end_date, None)
```

Comment:
This test only checks if the log entry is created. We don't want to check the ShareOwnerships here, it is done in `test_membershipResignationDeleteView_default_sharesEndDateSetToNone`